app/assets/stylesheets/components/_navigation.scss

```diff
@@ -2,26 +2,22 @@
   @extend .unstyled-list;
   text-align: right;
   margin-top: 22px;
+  padding-right: $baseline-unit;
 }
 
 %navigation-item {
   @include inline-block;
   line-height: 1;
-  border-right: 1px solid $color-primary-nav;
-  padding: 0 $baseline-unit*1.5 0 $baseline-unit*0.7;
+  border-left: 1px solid $color-primary-nav;
+  padding: 0 $baseline-unit*0.7 0 $baseline-unit*1.5;
```

Comment:
Yeah, I agree. Maybe I can use (r)ems. Baseline is semantically a vertical anyway right?